py/server/tests/test_numba_vectorized_column.py

```diff
@@ -22,9 +22,9 @@ def test_part_of_expr(self):
             t = empty_table(10).view(formulas=["I=ii", "J=(ii * 2)"]).update("K = 2 * vectorized_func(I, J)")
 
     def test_cast(self):
-        t = empty_table(10).view(formulas=["I=ii", "J=(ii * 2)"]).update("K = (float)vectorized_func(I, J)")
-        html_output = to_html(t)
-        self.assertIn("<td>9</td>", html_output)
+        with self.assertRaises(DHError) as cm:
```

Comment:
Why doesn't this work now, and can we add a test that should?

Reply:
it never worked, we simply ignored the casting.